spec/tools/benchmarking/memusg_python

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+import sys
+import os
+from time import sleep
+from subprocess import *
+
+# -o file.txt : output information to a file rather than stderr
+# -d          : debug mode
+
+
+out = sys.stdout
+DEBUG = False
+
+child_args = []
+i = 1
+while i < len(sys.argv):
+    if sys.argv[i] == '-o':
+        i += 1
+        out = open(sys.argv[i], 'w')
+    elif sys.argv[i] == '-d':
+        DEBUG = True
+    else:
+        child_args.append(sys.argv[i])
+    i += 1
+
+# child_command should be a single argument as if to "/bin/sh -c 'child_command'"
+# when shell=True is enabled
+child_command = ' '.join(child_args)
+
+def log(msg):
+    if DEBUG:
+        print >>sys.stderr, "memusg: {}".format(msg)
+
+def get_vsize(sid):
+    vsize = 0
+    # Example: /bin/ps -o vsize= --sid 23928
```

Comment:
you changed the ps command s/vsize/rssize/ right?
please update function name, output message "vmpeak" to e.g. "peak rssize" (and this variable & comment).